### PR TITLE
Add error logging in as-http example

### PR DESCRIPTION
**Pull request by anson627**
Update as-http example with error logging
Useful for reproducing and debugging as-http/tchannel error
@ShanniLi @Raynos @davewhat

### CODE REVIEW

File: node/examples/http_egress.js [hunk 1]
```diff
@@ -87,5 +87,9 @@ function onHTTPListening() {
 }
 
 function onHTTPRequest(hreq, hres) {
-    asHTTP.forwardToTChannel(svcChan, hreq, hres, {}, function() { return true; });
+    asHTTP.forwardToTChannel(svcChan, hreq, hres, {}, function(error) {
```
Comment (ShanniLi) [inline node/examples/http_egress.js]: it would be nice to also add name to the function

File: node/examples/http_ingress.js [hunk 2]
```diff
@@ -86,5 +86,9 @@ function onRequest(inreq, outres) {
     asHTTP.forwardToHTTP(svcChan, {
         host: destHost,
         port: destPort,
-    }, inreq, outres, function() { return true; });
+    }, inreq, outres, function(error) {
```
Comment (ShanniLi) [inline node/examples/http_ingress.js]: it would be nice to also add name to the function

**Comment by ShanniLi** [comment]
lgtm with unblocking comments

File: node/examples/http_egress.js [hunk 3]
```diff
@@ -87,5 +87,9 @@ function onHTTPListening() {
...
+        if (error) {
+            console.log('forward to tchannel failed', error);
```
Comment (Raynos) [inline node/examples/http_egress.js]: Never console.log. Always self.logger

Comment (anson627) [inline node/examples/http_egress.js]: This is example code. Do we want logtron here?

Comment (Raynos) [inline node/examples/http_egress.js]: Oh sorry; let's use `console.error` then.

**Pull request merged**